detectable/src/main/java/com/synopsys/integration/detectable/detectables/clang/dependencyfile/DependenyListFileParser.java

```diff
@@ -61,9 +63,25 @@
         logger.trace(String.format("dependencies, backslashes removed: %s", depsListString));
 
         final String[] deps = depsListString.split("\\s+");
+        final List<String> depsList = new ArrayList<>(deps.length);
         for (final String includeFile : deps) {
-            logger.trace(String.format("\t%s", includeFile));
+            if (StringUtils.isNotBlank(includeFile)) {
+                depsList.add(toCanonical(includeFile));
+            }
         }
-        return Arrays.asList(deps);
+        return depsList;
+    }
+
+    private String toCanonical(final String rawPath) {
+        final File targetFile = new File(rawPath);
+        final String canonicalPath;
```

Comment:
Could you change this to 
`String canonicalPath = rawPath;
        try {
            canonicalPath = targetFile.getCanonicalPath();
        } catch (final IOException e) {
            logger.warn(String.format("Unable to convert %s to canonical path", rawPath));
        }
        logger.trace(String.format("Canonicalized %s to %s", rawPath, canonicalPath));
        return canonicalPath;`